src/handlers/dashboard/get_chart_data.py

```diff
@@ -22,24 +26,26 @@ def _get_table_cols(table_name: str, version: str | None = None) -> list:
     """
 
     s3_bucket_name = os.environ.get("BUCKET_NAME")
-    prefix = f"{BucketPath.CSVAGGREGATE.value}/{table_name.split('__')[0]}/{table_name}"
+    prefix = f"{enums.BucketPath.CSVAGGREGATE.value}/{dp_id.split('__')[0]}/{dp_id[:-4]}"
```

Comment:
What's with ignoring the last 4 characters? Also, is there a separator we can call `rsplit` on instead? Is this meant to skip the 3-character version and the underscore? Wouldn't you need to skip 5 for that, since there are two underscores?

Also, maybe save it in a variable since you use it again below and it'd be easy to change one but not the other (plus, it lets you give it a sensible name, which might avoid questions like "what's with ignoring the last 4 characters")

Reply:
hm - yeah, test data artifact, will bump out to five or flip to looking for last dunder.